src/allmydata/util/encodingutil.py

```diff
@@ -179,8 +186,15 @@ def is_printable_ascii(s):
 def unicode_to_output(s):
     """
     Encode an unicode object for representation on stdout or stderr.
+
+    On Python 3 just returns the string unchanged, since it is not necessary to
+    encode in any way.
```

Comment:
Hm.  It's more like "the file object the output is written to is responsible for the encoding" right?  (which is definitely good, just noting that encoding still happens, we just don't have to do it any earlier than the internals of file.write)